warehouse/legacy/api/json.py

```diff
@@ -217,3 +217,120 @@ def json_release_slash(release, request):
         ),
         headers=_CORS_HEADERS,
     )
+
+
+@view_config(
+    route_name="legacy.api.json.latest",
+    context=Project,
+    renderer="json",
+    decorator=_CACHE_DECORATOR,
+)
+def json_latest(project, request):
+    release = project.latest_version
+    version = release.version if release else None
```

Comment:
Stylistically/functionally, is this the best way to accommodate the possibility of `project.latest_version` being `None`?